soc/nxp/imxrt/soc.yml

```diff
@@ -38,6 +38,12 @@ family:
     - name: mimxrt685s
       cpuclusters:
       - name: cm33
+  - name: imxrt7xx
+    socs:
+    - name: mimxrt798s
+      cpuclusters:
+      - name: cpu0
```

Comment:
Same question here- I think we should name these CPU clusters differently given the presence of the HIFI DSP

Reply:
I agree with @danieldegrasse. 
We also have, on i.MXRT700, 2 DSPs: HiFi4 and HiFi1. We are going to name the two cores simply: hifi1 and hifi4.

Reply:
This is wrong, cpucluster is the type, see above, so cm33 is a cpu cluster, cpu0/cpu1 is not a cluster

Reply:
Updated to cm33_cpu0/cm33_cpu1. Thank you.